src/main/resources/api/service_openapi.yaml

```diff
@@ -1,264 +1,237 @@
+      tags: [Unauthenticated]
       responses:
-        200:
-          $ref: '#/components/responses/VersionResponse'
+        '200':
+          description: System version response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SystemVersion'
 
-  '/api/workspaces/v1':
+  # Workspace paths
+  /api/workspaces/v1:
     post:
-      description: |
-        Create a new Workspace. This should only be called by Folder Manager.
+      summary: Create a new Workspace. This should only be called by Folder Manager.
       operationId: createWorkspace
-      tags:
-      - workspace
+      tags: [Workspace]
       requestBody:
-        description: |
-          Auth token, spend profile, and policy information from
-          Folder Manager, as well as async job information.
+        required: true
         content:
           application/json:
             schema:
               $ref: '#/components/schemas/CreateWorkspaceRequestBody'
       responses:
-        200:
-          description: Ok
-          $ref: '#/components/responses/CreateResponse'
-        400:
-          description: Bad request - invalid id, badly formed
-          $ref: '#/components/responses/ErrorResponse'
-        401:
-          description: Permission denied
-          $ref: '#/components/responses/ErrorResponse'
-        500:
-          description: Creation request error
-          $ref: '#/components/responses/ErrorResponse'
-  '/api/workspaces/v1/{id}':
+        '200':
+          $ref: '#/components/responses/CreatedWorkspaceResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/NotFound'
+
+  /api/workspaces/v1/{id}:
     parameters:
-    - $ref: '#/components/parameters/Id'
+      - $ref: '#/components/parameters/Id'
     get:
-      description: |
-        Get an existing Workspace.
+      summary: Get an existing Workspace.
       operationId: getWorkspace
-      tags:
-        - workspace
+      tags: [Workspace]
       responses:
-        200:
+        '200':
           description: OK
-          $ref: '#/components/responses/GetResponse'
-        500:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WorkspaceDescription'
+        '500':
           description: Get request error
-          $ref: '#/components/responses/ErrorResponse'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
     delete:
-      description: |
-        Delete a Workspace. This should only be called by the Folder Manager.
+      summary: Delete a Workspace. This should only be called by the Folder Manager.
       operationId: deleteWorkspace
-      tags:
-      - workspace
+      tags: [Workspace]
       requestBody:
```

Comment:
this line still throws a validation error: `DELETE operations cannot have a requestBody.` I realize that we're documenting what the code is actually doing, so we shouldn't change it - and that error seems to be against the spec. I wonder if there's anything we can do about it.

No changes necessary for this PR, I'm just thinking out loud.